server/auth/types/openid/openid_auth.ts

```diff
@@ -219,6 +219,27 @@ export class OpenIdAuthentication extends AuthenticationType {
       return true;
     }
 
+    return false;
+  }
+
+  async refreshAccessToken(
+    cookie: SecuritySessionCookie,
+    request: OpenSearchDashboardsRequest
+  ): Promise<string> {
+    if (
+      cookie.authType !== this.type ||
+      !cookie.username ||
+      !cookie.expiryTime ||
+      (!cookie.credentials?.authHeaderValue && !this.getExtraAuthStorageValue(request, cookie)) ||
+      !cookie.credentials?.expires_at
+    ) {
+      return '';
+    }
+
+    if (cookie.credentials?.expires_at > Date.now()) {
```

Comment:
With the check on 243, drop the coalescing operator?  I'm not sure what this check would resolve to if it wasn't set (for whatever crazy reason).  Could you also add unit tests?
```suggestion
    if (cookie.credentials.expires_at > Date.now()) {
```